### ISSUE TITLE
 i can't type in input

### ISSUE THREAD

**Issue by zeing** [craftpip/jquery-confirm#169]
after i submit button from  form in modal and 
use   "event.preventDefault(); " to stop refresh 
and then it will show $confirm which have  1 input(remark) for confirm form in modal 
![image](https://cloud.githubusercontent.com/assets/20161393/20340420/ff373514-ac13-11e6-99bd-b811264b18d3.png)
 but !! i can't type text in input ( i try in input and textarea ) 

![image](https://cloud.githubusercontent.com/assets/20161393/20340530/834709ec-ac14-11e6-8fb4-da10e9a19f19.png)
help me pls
thank you

**Comment by craftpip** [comment]
hello @zeing 
I will need more information to figure out the problem, please post the code or link to test it will be really helpful.

**Comment by dvmobley** [comment]
sounds like a z-index problem...inspect the document. Is there anything overlaying your Alert?

**Comment by zeing** [comment]
i know it will don't work  when modal behind it .  ( my modal can use ESC for closing  and when  it can type in input ) . 
![image](https://cloud.githubusercontent.com/assets/20161393/20352621/788b7d40-ac4a-11e6-87aa-b0c313a9a92c.png)

![image](https://cloud.githubusercontent.com/assets/20161393/20352640/85ba1440-ac4a-11e6-8eee-be3629238cd1.png)


 @dvmobley  i set z-index for input it don't work 


wait a min. for my code

**Comment by zeing** [comment]
i want to save form but i want to type remark before submit . i want type remark from jquery-confirm 
this is my validator and submit form  and  manual to send data by ajax

this is my code 

show form in modal and set validation 
`
    $('#modalBranch').on('shown.bs.modal', function () {
        $('#formData').validator({custom: fvcustom}).on('submit',function (event) {
            event.preventDefault();
             $.confirm({
                title: 'Alert alert!',
                content: '<input type="text" class="form-control" >',
                icon: 'fa fa-rocket',
                animation: 'zoom',
                closeAnimation: 'zoom',
                buttons: {
                    okay: {
                        text: 'Okay',
                        btnClass: 'btn-primary'
                    }
                }
            });
        });
        $("#saveBranch").addClass('disabled');
        ResizeMap();
    });

    $('#modalBranch').on('hide.bs.modal', function () {
        $('#formData').off('submit').validator('destroy');
    });

`

thank you

**Comment by dvmobley** [comment]
Why are you using an external function to do form validation? Why not put it in to the action event of the button. See http://craftpip.github.io/jquery-confirm/#button-functions for an example.


Dale Mobley
IT Administrator
UTC Housing
O – 423.425.5908

Solving Problems One at A Time…

**Comment by dvmobley** [comment]
Can we access your example?

Dale Mobley
IT Administrator
UTC Housing
O – 423.425.5908

Solving Problems One at A Time…

**Comment by zeing** [comment]
because i want to realtime validate form before submit when my form invalid  it can't click submit button . 

 sorry i coding in nodejs  , i will make example in https://jsfiddle.net/

**Comment by zeing** [comment]
sorry if this is a bad example , this is same my problem  
http://jsfiddle.net/zeing/h3WDq/1721/

**Comment by dvmobley** [comment]
I think the problem is that you are mixing two different types of modals. I replaced your modal with a jquery-confirm confirm object and everything works as it should.

http://jsfiddle.net/h3WDq/1723/


Dale Mobley
IT Administrator
UTC Housing
O – 423.425.5908

Solving Problems One at A Time…

**Comment by zeing** [comment]
okay i should use one thing . thank you very much

**Comment by craftpip** [comment]
Thanks @dvmobley 👍

**Comment by zeing** [comment]
sorry for my english :D

**Comment by craftpip** [comment]
apparently this was caused by tabindex in 
```html
<div id="myModal1" class="modal hide" tabindex="-1" role="dialog">
```
removing it solved the issue, maybe because jconfirm has the same value too.
i will look into this. 

Thanks for reporting ! 👍

**Comment by zeing** [comment]
Okay i will try

Sent from my iPhone

On 17 Nov 2016, at 01:28, Boniface Pereira <notifications@github.com<mailto:notifications@github.com>> wrote:


Closed #169<https://github.com/craftpip/jquery-confirm/issues/169>.

-

**Comment by zeing** [comment]
I have tried already
It work !!!
Thank you very much

Sent from my iPhone

On 17 Nov 2016, at 01:27, Boniface Pereira <notifications@github.com<mailto:notifications@github.com>> wrote:


apparently this was caused by tabindex in

<div id="myModal1" class="modal hide" tabindex="-1" role="dialog">

removing it solved the issue, maybe because jconfirm has the same value too.
i will look into this.

Thanks for reporting ! ?

-
